### DISCUSSION TITLE
calibration

### DISCUSSION THREAD

**Discussion by pashameow** [AveYo/ShowMMR#5] [Q&A] [answered]
is it showing mmrs on calibration?

**Comment by AveYo** [comment, marked as answer]
No, that's not exposed by Valve. 
_(it was exposed for a brief period in the past, but that was a "bug" and got fixed)_

**Comment by pashameow** [comment]
I would bet,  the melonity cheat has a function for displaying mmr on calibration(I know that it is supposedly forbidden, but I have not received any bans)21.06.2023, 19:48, "AveYo" ***@***.***>:
No, that's not exposed by Valve.
(it was exposed for a brief period in the past, but that was a "bug" and got fixed)
